project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/c4m_content_photo.module

```diff
@@ -14,16 +14,74 @@ function c4m_content_photo_theme_registry_alter(&$theme_registry) {
 }
 
 /**
+ * Implements hook_node_presave().
+ *
+ * If a photo was uploaded and if is part of a photo album rename Album name
+ * along with a numerical suffix.
+ */
+function c4m_content_photo_node_presave($node) {
```

Comment:
@amitaibu 
Why aren't we using `hook_node_insert` in here instead of the `presave` one ?